runhouse/servers/cluster_servlet.py

```diff
@@ -433,3 +431,7 @@ async def asend_cluster_logs_to_den(self):
 
     def send_cluster_logs_to_den(self):
         asyncio.run(self.asend_cluster_logs_to_den())
+
+    def set_cluster_logger(self, logs_level: str = DEFAULT_LOG_LEVEL):
```

Comment:
a little confused by this (also related to my previous comment about initialize a global logger object), instead of re-initializing the logger why can't we just reset its level?